src/redux/book/book.js

```diff
@@ -0,0 +1,42 @@
+const ADD_A_BOOK = 'bookStore/book/ADD_A_BOOK';
+const REMOVE_A_BOOK = 'bookStore/book/REMOVE_A_BOOK';
+const LIST_BOOKS = 'bookStore/book/LIST_BOOKS';
+
+const initialState = [{
+  id: '1', title: 'bealy', author: 'father', category: 'java',
+}];
```

Comment:
- [ ] Kindly set the `intialState` with an empty array of books as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Set%20the%20initial%20state%20to%20be%20an%20empty%20array%20of%20books.).